manage.py

```diff
@@ -72,6 +72,9 @@ def add_fake_data(number_users):
     User.generate_fake(count=number_users)
     Resource.generate_fake()
 
+@manager.command
+def populate_data():
```

Comment:
In keeping with the manager command called `add_fake_data`, can this one be called something like `add_seattle_data`?